warehouse/templates/includes/file-details.html

```diff
@@ -147,16 +155,26 @@ <h3>Provenance</h3>
                 <li>Owner: <a href="{{ viewer.owner }}">{{ viewer.owner }}</a></li>
                 <li>Access: <code>{{ viewer.access }}</code></li>
               </ul>
+              {% endif %}
               Publication detail:
               <ul>
                 <li>Token Issuer: <code>{{ viewer.issuer }}</code></li>
+                {% if viewer.publisher.kind == 'Google' %}
```

Comment:
I dislike that we use both single- and double-quoted strings in our `if` statements. No change necessary here, but one day I'll change our linter and do a mass cleanup...